app/main.development.js

```diff
@@ -12,6 +12,9 @@ if (process.env.NODE_ENV === 'production') {
 
 if (process.env.NODE_ENV === 'development') {
   require('electron-debug')(); // eslint-disable-line global-require
+  const path = require('path'); // eslint-disable-line
+  const p = path.join(__dirname, '..', 'static', 'node_modules'); // eslint-disable-line
+  require('module').globalPaths.push(p); // eslint-disable-line
```

Comment:
Found [another project following the same work flow this way](https://github.com/sqlectron/sqlectron-gui/blob/master/src/browser/main.js#L11) from [this conversation](https://github.com/electron-userland/electron-builder/issues/326)